Source/Core/getStringFromTypedArray.js

```diff
@@ -0,0 +1,49 @@
+/*global define*/
+define([
+        './defined',
+        './DeveloperError'
+    ], function(
+        defined,
+        DeveloperError) {
+    "use strict";
+    /*global TextDecoder*/
+
+    /**
+     * @private
+     */
+    var getStringFromTypedArray = function(buffer, byteOffset, length) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(buffer)) {
+            throw new DeveloperError('buffer is required.');
+        }
+
+        if (!defined(byteOffset)) {
+            throw new DeveloperError('byteOffset is required.');
+        }
+
+        if (!defined(length)) {
+            throw new DeveloperError('length is required.');
+        }
+        //>>includeEnd('debug');
+
+        return getStringFromTypedArray.decode(new Uint8Array(buffer, byteOffset, length));
+    };
+
+    // Exposed functions for testing
+    getStringFromTypedArray.decodeWithTextDecoder = function(view) {
+        var decoder = new TextDecoder('utf-8');
+        return decoder.decode(view);
+    };
+
+    getStringFromTypedArray.decodeWithFromCharCode = function(view) {
+        return String.fromCharCode.apply(String, view);
```

Comment:
Hi, 
I also found a problem with line 39. I used a bgltf model with a json size of approximately 200kb. With Safri/Ipad I encountered a "Maximum call stack size exceeded error" and the model doesn't load. Safari has a smaller maximum stack size than other browsers. When i change the code to:

     var result = "";
        for(var i = 0; i < view.length; i++){
          result += String.fromCharCode(view[i]);
        }
        return result

The model loads fine. Otherwise the binary-gltf support works pretty good.